src/pages/user/profile.tsx

```diff
@@ -1,5 +1,68 @@
+import { useSession, useSupabaseClient } from '@supabase/auth-helpers-react';
+import { useEffect, useState } from 'react';
+
 const Profile = () => {
-  return <div>Profile Setting</div>;
+  const [bio, setBio] = useState('');
+  const [newBio, setNewBio] = useState('');
+
+  const supabase = useSupabaseClient();
+  const session = useSession();
+
+  useEffect(() => {
+    if (session === null) return;
+
+    console.log('not done');
```

Comment:
remove later?

Reply:
well yeah this whole page will be removed later cuz veer is supposed to make a better one